R/common-error-checking-utils.R

```diff
@@ -0,0 +1,24 @@
+#' @title 
```

Comment:
Let's call this `isTargetYN` and change it to 
- output TRUE if all values are Y or N. 
- If there are NAs, still output true, but print a warning "The target column contains NA values. Healthcareai will remove them in `funcitonName`."
- If there is anything but Y, N, or NA, return FALSE.